### DISCUSSION TITLE
Disabling the Dependency Dashboard

### DISCUSSION THREAD

**Discussion by rvillane** [renovatebot/renovate#31221] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub Enterprise Cloud (github.com)

### Please tell us more about your question or problem

I tried following the advice provide here to disable the dependency dashboard -> https://docs.renovatebot.com/key-concepts/dashboard/#how-to-disable-the-dashboard and set `dependencyDashboard` property to false and closed the currently opened Dependency Dashboard issues in GitHub.com and the next bot execution reopened all the Dep Dashboard issues again.

Here is the configuration I'm using:
```
module.exports = {
  endpoint: 'https://api.github.com',
  platform: 'github',
  onboarding: true,
  onboardingConfig: {
    extends: ['config:base'],
  },
  dependencyDashboard: false,
  autodiscover: false,
  repositories: [
    'MyOrg/Myrepo',
  ],
  requireConfig: "required",
  automerge: false,
  extends: ['group:allNonMajor'],
  enabledManagers: ['npm','dockerfile','terraform'],
  rangeStrategy: 'pin',
  recreateClosed: true,
  minimumReleaseAge: "7 days",
  prCreation:'not-pending',
  packageRules: [
    {
      matchDatasources: ['docker'],
      prCreation: 'immediate',
    },
    {
      "matchPackageNames": ["node"],
      "allowedVersions": "<21"
    },
  ],
  lockFileMaintenance: { enabled: true },
  bumpVersion: 'patch',
  labels: [
    'renovate'
  ]
};
```

I would expect that if the Dep Dashboard is disabled, the GH issues are no longer needed.  Any idea what could be happening here? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rvillane** [comment]
In case it helps, enclosed you will find an obfuscated log from the last execution
[renovate-log.txt](https://github.com/user-attachments/files/16881785/renovate-log.txt)

**Reply by RahulGautamSingh** [reply]
From the logs:
```
DEBUG: Repository config (repository=MyOrg/nodejs-hello)
       "fileName": "renovate.json",
       "config": {
         "$schema": "https://docs.renovatebot.com/renovate-schema.json",
         "extends": ["config:base"]
       }
```

Its a conflict between repo and admin config. In the repo config `extends: [config:base]` enables dashboard, while in the admin config you explicitly disable it. 

Repo config is used over admin config hence dashboard is created. 
You will need to disable dashboard in repo config as well.

**Reply by rvillane** [reply]
Hi @RahulGautamSingh, can you please confirm if you mean something like this:

```
module.exports = {
  endpoint: 'https://api.github.com',
  platform: 'github',
  onboarding: true,
  onboardingConfig: {
    extends: ['config:base', ':disableDependencyDashboard'],
  },
  dependencyDashboard: false,
  .....
}

**Reply by RahulGautamSingh** [reply, marked as answer]
Also do the same for the repos that have already been onboarded.

**Reply by rvillane** [reply]
@RahulGautamSingh , thanks for the help, I can confirm your suggestion works as expected.
